client/state/data-layer/wpcom/read/lists/index.js

```diff
@@ -57,11 +58,10 @@ registerHandlers( 'state/data-layer/wpcom/read/lists/index.js', {
 						} ),
 					];
 				}
-				// NOTE: Add better handling for unexpected response format here.
```

Comment:
Removed this comment because it's a very rare edge case - the API will return a 4xx response if anything goes wrong with list creation and `onError` will handle that.